docs/notes-pa.md

```diff
@@ -3,4 +3,166 @@ how the permutation argument is formed.
 
 
 Permutation argument
-==============
\ No newline at end of file
+==============
+
+Within PLONK, the permutation 
```

Comment:
I think this sentence is a bit vague; can you expand or be a bit more concise on what you mean by "form arguments about the proving system"?